### DISCUSSION TITLE
My project on the official OpenDeck 2

### DISCUSSION THREAD

**Discussion by Jozio611** [shanteacontrols/OpenDeck#268] [Show and tell]
My project on the official OpenDeck 2 board, in a 1u rack housing, 31 switches and 11 potentiometers. I have a question, I want to program and build a new controller on Arduino Mega 2560, with Windows and a USBasp programmer, but I can't flash this board, could someone help me, Thank you
![OpenDeck V2](https://github.com/shanteacontrols/OpenDeck/assets/105865133/0fbc9261-ed05-4904-bbc6-5d07555d362f)

**Comment by Jozio611** [comment]
I will add that this controller works with the vsti arranger, and it works very well with it, sorry for my English, I use a translator, Greetings from Poland

**Comment by paradajz** [comment]
Looks really good, thanks for sharing.

**Comment by Jozio611** [comment]
![Zrzut ekranu (2)](https://github.com/shanteacontrols/OpenDeck/assets/105865133/3d42e993-b94a-4d06-96c0-d257e933bb0e)

Now I'm trying to build a new controller with Arduino Mega 2560, I finally managed to flash it, it turned out that the USBASP programmer was causing problems, I bought a new AVRISP MK2 programmer and everything is fine, everything works just like on the official OpenDeck v2 board. Paradajz, thank you for your work, you are GREAT!!!

**Reply by paradajz** [reply]
Good to know, and thank you. That looks like fairly old firmware version, though.

**Comment by Jozio611** [comment]
At the moment, v6.0.0 is sufficient for me, because I have it in the official OpenDeck v2 and it works very well. After many attempts, I finally managed to flash two systems in Arduino Mega, at this point I'm afraid to do anything once it works :)

**Reply by paradajz** [reply]
Fair enough!

**Reply by Jozio611** [reply]
I still have questions: which software version is the most stable at the moment? is it possible to flash Arduino without any risk? does the resistor have to be on single-color diodes? Thank you. Good night

**Reply by paradajz** [reply]
* They are all stable and tested.
* Not sure what kind of risk are you talking about
* Which resistor? Which diodes? I don't understand

**Comment by Jozio611** [comment]
![Zrzut ekranu (3)](https://github.com/shanteacontrols/OpenDeck/assets/105865133/d03c273a-bd5d-4974-b9c7-ac74a3e16e4c)

**Reply by paradajz** [reply]
Ah, the LEDs. Yes you do need the resistors on an Arduino.

**Reply by Jozio611** [reply]
Dziękuję za informację
